services/todo-legacy/datasources.ts

```diff
@@ -0,0 +1,12 @@
+export const datasources: Object = {
```

Comment:
In my experience, using a type definition in a place like this makes it easier to discover problems caused by the upstream code renaming properties. E.g. if you say `dataSources` should conform to `DataSourceConfig` interface, and then we decide to rename a property somewhere inside the config structure, then `tsc` should be able to point that problem to you.